jenkins/README.md

```diff
@@ -0,0 +1,14 @@
+#### Using Jenkins to test playbook2image
+
+
+```
+oc process -f jenkins-ephemeral-template.yaml MEMORY_LIMIT=2Gi | oc create -f -
```

Comment:
Do we need a custom Jenkins template? Could we use the stock s2i jenkins template with our plugins.txt?